frontend/website/src/Render.re

```diff
@@ -107,14 +107,13 @@ module Router = {
 };
 
 let jobOpenings = [|
-  ("engineering-manager", "Engineering Manager (San Francisco)."),
   ("product-manager", "Product Manager (San Francisco)."),
-  ("senior-frontend-engineer", "Senior Frontend Engineer (San Francisco)."),
-  ("protocol-engineer", "Senior Protocol Engineer (San Francisco)."),
   (
-    "director-of-business-development",
-    "Director of Business Development (San Francisco).",
+    "senior-frontend-engineer",
```

Comment:
maybe this can also say senior-product-engineer

Reply:
I think we need to keep this `senior-frontend-engineer.html` so we don't have dead links elsewhere (I guess the right solution would be to have both or a redirect, but that would be more eng effort)